app/src/org/commcare/android/resource/installers/ProfileAndroidInstaller.java

```diff
@@ -119,7 +119,7 @@ public boolean install(Resource r, ResourceLocation location, Reference ref,
         } catch (XmlPullParserException | InvalidReferenceException | IOException e) {
             e.printStackTrace();
         } catch (InvalidStructureException e) {
-            throw new UnresolvedResourceException(r, "Invalid content in the Profile Definition: " + e.getMessage(), true);
```

Comment:
Hey @shubham1g5 I believe this is the result of the fact that the profile file (being the first entry point for the app) can sometimes not point to a profile file at all due to a captive portal or similar. I'm not sure what the downstream implications of changing this would be, there, but wanted to surface it in case it jogged any memories

Reply:
I see, still makes sense to me to classify it as `InvalidResourceException`. User is going to see the same error message in both cases and there is no difference in downstream handeling of `UnresolvedResourceException` vs `InvalidResourceException`. The change just helps in better classifying the errors for Analytics purpose.